.github/workflows/release.yaml

```diff
@@ -9,26 +9,27 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - name: Set up Python
-        uses: actions/setup-python@v4
+      - name: Install uv and set the python version
+        uses: astral-sh/setup-uv@v5
         with:
-          python-version: '3.8'
+          python-version-file: ".python-version"
+          enable-cache: true
 
-      - name: Install dev dependencies
-        run: python -m pip install --upgrade pip setuptools poetry
+      - name: Install the project
+        run: uv sync
 
       - name: Build binary wheel and a source tarball
-        run: poetry build
+        run: uv build --package vatvalidate --out-dir dist
 
-      - name: Get version
-        run: echo "VERSION=$(poetry version --short)" >>$GITHUB_OUTPUT
+      - name: Get new version
+        run: echo "VERSION=$(uvx --from toml-cli toml get --toml-path pyproject.toml project.version)" >>$GITHUB_OUTPUT
```

Comment:
do you want to setup the calver stuff from the other repos? it just saves having to remember changing versions faff